ncurses6: 20171125 -> 6.1

dtzWill opened the pull request:
Release notes are currently available here: https://www.gnu.org/software/ncurses/#h2-release-notes
(that URL is not future-proof, for what it's worth).

More verbose changelog:
https://invisible-island.net/ncurses/NEWS.html#t20180127

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

From my experience on slightly older tree,
a few *old* programs incorrectly rely on ncurses internal details
and may now require setting NCURSES_INTERNAL=1 to allow this badness.

dtzWill commented:
@GrahamcOfBorg build ncurses6 vim

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34477&attempt_id=none)

<details><summary>Partial log (click to expand)</summary><p>

```
stripping (with command strip and flags -S) in /nix/store/mmwh9zaz40b0wk1jilag38qw33g9f8pf-vim-8.0.1428/bin 
patching script interpreter paths in /nix/store/mmwh9zaz40b0wk1jilag38qw33g9f8pf-vim-8.0.1428
/nix/store/mmwh9zaz40b0wk1jilag38qw33g9f8pf-vim-8.0.1428/bin/vimtutor: interpreter directive changed from " /bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
/nix/store/mmwh9zaz40b0wk1jilag38qw33g9f8pf-vim-8.0.1428/share/vim/vim80/macros/less.sh: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
basename: invalid option -- 'w'
Try 'basename --help' for more information.
/nix/store/mmwh9zaz40b0wk1jilag38qw33g9f8pf-vim-8.0.1428/share/vim/vim80/tools/ref: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
/nix/store/mmwh9zaz40b0wk1jilag38qw33g9f8pf-vim-8.0.1428/share/vim/vim80/tools/vimm: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
/nix/store/mmwh9zaz40b0wk1jilag38qw33g9f8pf-vim-8.0.1428/share/vim/vim80/tools/vimspell.sh: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
checking for references to /tmp/nix-build-vim-8.0.1428.drv-0 in /nix/store/mmwh9zaz40b0wk1jilag38qw33g9f8pf-vim-8.0.1428...
```
</p></details>

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34477&attempt_id=none)

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/7rqf33qmibpxynn4mh0fa384s6x9s652-vim-8.0.1428/share/vim/vim80/tools/vimspell.sh: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
/nix/store/7rqf33qmibpxynn4mh0fa384s6x9s652-vim-8.0.1428/share/vim/vim80/tools/vimm: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
/nix/store/7rqf33qmibpxynn4mh0fa384s6x9s652-vim-8.0.1428/share/vim/vim80/tools/ref: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
basename: invalid option -- 'w'
Try 'basename --help' for more information.
/nix/store/7rqf33qmibpxynn4mh0fa384s6x9s652-vim-8.0.1428/share/vim/vim80/macros/less.sh: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
/nix/store/7rqf33qmibpxynn4mh0fa384s6x9s652-vim-8.0.1428/bin/vimtutor: interpreter directive changed from " /bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
checking for references to /build in /nix/store/7rqf33qmibpxynn4mh0fa384s6x9s652-vim-8.0.1428...
/nix/store/hah7sz6bn1g80p0nfacipfj2da8hr1i2-ncurses-6.1
/nix/store/7rqf33qmibpxynn4mh0fa384s6x9s652-vim-8.0.1428
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34477&attempt_id=none)

<details><summary>Partial log (click to expand)</summary><p>

```
cannot build derivation '/nix/store/rnj8ydi89gsbq2ds51lsmq4friwa089k-apple-framework-CoreVideo.drv': 8 dependencies couldn't be built
cannot build derivation '/nix/store/ckj2q33psiwdis7ca3swl0409a8w22qc-apple-framework-Foundation.drv': 9 dependencies couldn't be built
cannot build derivation '/nix/store/f3myyif18scxhar30bh44pvvxyw65gq0-apple-framework-QuartzCore.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/ij4g8s0ynfcm90rdjizabjxfi7cpa8ci-apple-framework-Carbon.drv': 10 dependencies couldn't be built
cannot build derivation '/nix/store/25smldpvjl6f4p475ksjk9mc3lf78z9v-apple-framework-AudioUnit.drv': 6 dependencies couldn't be built
cannot build derivation '/nix/store/pa10246wfnx9axhvy96i7i28573mabj8-apple-framework-AudioToolbox.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/jm7n8qqpm1z63xnzmbm83gf3nfz1iaz6-apple-framework-AppKit.drv': 6 dependencies couldn't be built
cannot build derivation '/nix/store/3bii6795vx795kxal9y361rk5lb59jh7-apple-framework-Cocoa.drv': 4 dependencies couldn't be built
cannot build derivation '/nix/store/iczy5577gja6vy6d15i2ghqh7qa80fpm-vim-8.0.1428.drv': 7 dependencies couldn't be built
error: build of '/nix/store/avj599l8n2fjbrivf6iwv78blh535ria-ncurses-6.1.drv', '/nix/store/iczy5577gja6vy6d15i2ghqh7qa80fpm-vim-8.0.1428.drv' failed
```
</p></details>

dtzWill commented:
(Darwin failure seems rather unrelated AFAICT)

dtzWill commented:
@grahamcofborg build ncurses6 vim

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34477&attempt_id=262d7430-45c9-4546-acbf-2ad977a8eac0)

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451/share/vim/vim80/tools/vimspell.sh: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451/share/vim/vim80/tools/vimm: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451/share/vim/vim80/tools/ref: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
basename: invalid option -- 'w'
Try 'basename --help' for more information.
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451/share/vim/vim80/macros/less.sh: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451/bin/vimtutor: interpreter directive changed from " /bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
checking for references to /build in /nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451...
/nix/store/hah7sz6bn1g80p0nfacipfj2da8hr1i2-ncurses-6.1
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451
```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34477&attempt_id=bfb30723-78bf-4df2-ac14-31de6430a262)

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451/bin/vimtutor: interpreter directive changed from " /bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
basename: invalid option -- 'w'
Try 'basename --help' for more information.
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451/share/vim/vim80/tools/vimm: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451/share/vim/vim80/tools/ref: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451/share/vim/vim80/tools/vimspell.sh: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451/share/vim/vim80/macros/less.sh: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
checking for references to /tmp/nix-build-vim-8.0.1451.drv-0 in /nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451...
/nix/store/5a5f5d604p99iala1yrcgn1nw6al6rs9-ncurses-6.1
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34477&attempt_id=0e50548b-2aa3-4f46-ad9a-f1d27f8b0501)

<details><summary>Partial log (click to expand)</summary><p>

```
cannot build derivation '/nix/store/c4zk3vlpnw0dcw5439dxhaz6rypzrlrj-apple-framework-CoreVideo.drv': 8 dependencies couldn't be built
cannot build derivation '/nix/store/6myagpzjvwhdd30ddvan391h848gr1pa-apple-framework-Foundation.drv': 9 dependencies couldn't be built
cannot build derivation '/nix/store/22zcmi16la6dj0rxl8sy8908dahd561c-apple-framework-QuartzCore.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/d154srfcqb9qfs5p8c3fjxhc5xp961v2-apple-framework-Carbon.drv': 10 dependencies couldn't be built
cannot build derivation '/nix/store/jizm0c82ipca4b3dzv81fclv3v2l7bm3-apple-framework-AudioUnit.drv': 6 dependencies couldn't be built
cannot build derivation '/nix/store/c361zbhc69k75riw3vw48mkak7bg0hdk-apple-framework-AudioToolbox.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/yvvmxbzlplab68n63jhi6zbd0shpfrv2-apple-framework-AppKit.drv': 6 dependencies couldn't be built
cannot build derivation '/nix/store/1g0jf4mwm0wvflmi32j8fxxa5nzwzf2j-apple-framework-Cocoa.drv': 4 dependencies couldn't be built
cannot build derivation '/nix/store/d2wkyap93zb0rr04sxind5nhj001m3g2-vim-8.0.1451.drv': 7 dependencies couldn't be built
error: build of '/nix/store/d2wkyap93zb0rr04sxind5nhj001m3g2-vim-8.0.1451.drv', '/nix/store/inwkjcdxw1rfbmi1dajf4pzh28k6yshk-ncurses-6.1.drv' failed
```
</p></details>

Mic92 commented:
@GrahamcOfBorg build ncurses6 vim

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34477&attempt_id=7fcc98d5-c9ed-4980-b7fa-eeb84c5015b2)

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451/share/vim/vim80/tools/vimspell.sh: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451/share/vim/vim80/tools/vimm: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451/share/vim/vim80/tools/ref: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
basename: invalid option -- 'w'
Try 'basename --help' for more information.
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451/share/vim/vim80/macros/less.sh: interpreter directive changed from "/bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451/bin/vimtutor: interpreter directive changed from " /bin/sh" to "/nix/store/7fxbh1yhagvwbdrmdyyy5ghcjhwjndhs-bash-4.4-p12/bin/sh"
checking for references to /build in /nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451...
/nix/store/hah7sz6bn1g80p0nfacipfj2da8hr1i2-ncurses-6.1
/nix/store/ggdvscmcjqj0divhsxpycdr43xlzbkad-vim-8.0.1451
```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34477&attempt_id=e58d5e0a-6518-4787-9f83-e15f87e1fba2)

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451/bin/vimtutor: interpreter directive changed from " /bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
basename: invalid option -- 'w'
Try 'basename --help' for more information.
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451/share/vim/vim80/tools/vimm: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451/share/vim/vim80/tools/ref: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451/share/vim/vim80/tools/vimspell.sh: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451/share/vim/vim80/macros/less.sh: interpreter directive changed from "/bin/sh" to "/nix/store/yq03c2ny43mc24j7dq5riznzb09ddhpq-bash-4.4-p12/bin/sh"
checking for references to /tmp/nix-build-vim-8.0.1451.drv-0 in /nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451...
/nix/store/5a5f5d604p99iala1yrcgn1nw6al6rs9-ncurses-6.1
/nix/store/17fk3q2y1dpvnz67yy8fr57rqxiwcyc3-vim-8.0.1451
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34477&attempt_id=c6bd3e27-1b5c-4024-99aa-d8ccc1e0791a)

<details><summary>Partial log (click to expand)</summary><p>

```
cannot build derivation '/nix/store/b092g5a5s5mz46bh53viy2jf5h52yc2g-apple-framework-CoreVideo.drv': 8 dependencies couldn't be built
cannot build derivation '/nix/store/pbfa6n5h29jm6hbb10dnc83f4n3xvl4a-apple-framework-Foundation.drv': 9 dependencies couldn't be built
cannot build derivation '/nix/store/9dm1ipn3ba8ys66vnla4w0lvksmzjl89-apple-framework-QuartzCore.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/i2v5gd5ja2bqy72bd7gsn5lrlf6kqq23-apple-framework-Carbon.drv': 10 dependencies couldn't be built
cannot build derivation '/nix/store/a5idd9fa6gzslmi45n518v98wqf86qaz-apple-framework-AudioUnit.drv': 6 dependencies couldn't be built
cannot build derivation '/nix/store/hq1vslnd310qcyfcwx66qdx8yz0y7dkp-apple-framework-AudioToolbox.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/za35mhldm8kqydjsqfzwy2did1nz8x5g-apple-framework-AppKit.drv': 6 dependencies couldn't be built
cannot build derivation '/nix/store/5n57cc6ii8k3kllq1683adpqdrgj177g-apple-framework-Cocoa.drv': 4 dependencies couldn't be built
cannot build derivation '/nix/store/isy35nhvyis1m1qns54n4h67gk8pzifg-vim-8.0.1451.drv': 7 dependencies couldn't be built
error: build of '/nix/store/i1swkr7hy4gf3l39c7fjlzqnzmkkjqwr-ncurses-6.1.drv', '/nix/store/isy35nhvyis1m1qns54n4h67gk8pzifg-vim-8.0.1451.drv' failed
```
</p></details>

dtzWill commented:
@GrahamcOfBorg build ncurses6 htop

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=13399994-bcbc-461c-9d2f-9863380ab9c1)

<details><summary>Partial log (click to expand)</summary><p>

```
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/pvs1a45xcb6pkl1s3zsmd7g978f45fyy-htop-2.1.0
shrinking /nix/store/pvs1a45xcb6pkl1s3zsmd7g978f45fyy-htop-2.1.0/bin/htop
gzipping man pages under /nix/store/pvs1a45xcb6pkl1s3zsmd7g978f45fyy-htop-2.1.0/share/man/
strip is /nix/store/5qj61lcvzlap87rf6blvf8p577d482bv-binutils-2.28.1/bin/strip
stripping (with command strip and flags -S) in /nix/store/pvs1a45xcb6pkl1s3zsmd7g978f45fyy-htop-2.1.0/bin 
patching script interpreter paths in /nix/store/pvs1a45xcb6pkl1s3zsmd7g978f45fyy-htop-2.1.0
checking for references to /tmp/nix-build-htop-2.1.0.drv-0 in /nix/store/pvs1a45xcb6pkl1s3zsmd7g978f45fyy-htop-2.1.0...
/nix/store/5a5f5d604p99iala1yrcgn1nw6al6rs9-ncurses-6.1
/nix/store/pvs1a45xcb6pkl1s3zsmd7g978f45fyy-htop-2.1.0
```
</p></details>

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=88524685-be72-4608-94bd-54ae27ce576a)

<details><summary>Partial log (click to expand)</summary><p>

```
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/kq8l0m4rl6rfj9fmnkcqfva6ndbz091s-htop-2.1.0
shrinking /nix/store/kq8l0m4rl6rfj9fmnkcqfva6ndbz091s-htop-2.1.0/bin/htop
gzipping man pages under /nix/store/kq8l0m4rl6rfj9fmnkcqfva6ndbz091s-htop-2.1.0/share/man/
strip is /nix/store/xmpjypwjmp2qi1chs5kr0hacnh161ls4-binutils-2.28.1/bin/strip
stripping (with command strip and flags -S) in /nix/store/kq8l0m4rl6rfj9fmnkcqfva6ndbz091s-htop-2.1.0/bin
patching script interpreter paths in /nix/store/kq8l0m4rl6rfj9fmnkcqfva6ndbz091s-htop-2.1.0
checking for references to /build in /nix/store/kq8l0m4rl6rfj9fmnkcqfva6ndbz091s-htop-2.1.0...
/nix/store/hah7sz6bn1g80p0nfacipfj2da8hr1i2-ncurses-6.1
/nix/store/kq8l0m4rl6rfj9fmnkcqfva6ndbz091s-htop-2.1.0
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=005dae57-410f-4fe8-977f-22af7c1a40ea)

<details><summary>Partial log (click to expand)</summary><p>

```
cannot build derivation '/nix/store/9a3ghpilrgc34fb12vsnj1487y0k085m-flex-2.6.4.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/h39kd0nkvwy04z5svpx6ixb1kxsgdkn7-CF-osx-10.10.5.drv': 8 dependencies couldn't be built
cannot build derivation '/nix/store/3c1hd0p7a1sr3sfqv6gmzciybnhqh1pl-bootstrap_cmds-dev-tools-7.0.drv': 4 dependencies couldn't be built
cannot build derivation '/nix/store/imdj66pwismabp7flw4z4j4lhfn5fjcx-python-2.7.14.drv': 14 dependencies couldn't be built
cannot build derivation '/nix/store/xaf387j2zagdjbi9kkd5ba2xplkbp3wv-xnu-osx-10.11.6.drv': 9 dependencies couldn't be built
cannot build derivation '/nix/store/3nflqhvdf16c9bjqm3p8yk0p2p2jpl34-IOKit-osx-10.11.6.drv': 3 dependencies couldn't be built
cannot build derivation '/nix/store/x7rkgsjgifjbss5m0xj1yzi6rmwr4xy7-configd-osx-10.8.5.drv': 9 dependencies couldn't be built
cannot build derivation '/nix/store/2174rpdralvv6i03fcsp9xp8d33az3w2-python-2.7.14.drv': 15 dependencies couldn't be built
cannot build derivation '/nix/store/0dfs3inj9ffr6h3w6qwsnv0s2n0f7375-htop-2.1.0.drv': 5 dependencies couldn't be built
error: build of '/nix/store/0dfs3inj9ffr6h3w6qwsnv0s2n0f7375-htop-2.1.0.drv', '/nix/store/yccxiwb5gyq290d42h7wraflxa3krxv0-ncurses-6.1.drv' failed
```
</p></details>

dtzWill commented:
repology stats on 6.1 adoption elsewhere: https://repology.org/metapackage/ncurses/information

dtzWill committed:
ncurses6: 20171125 -> 6.1

From my experience on slightly older tree,
a few *old* programs incorrectly rely on ncurses internal details
and may now require setting NCURSES_INTERNALS=1 to allow this badness.

Since this is release, we can grab it from gnu mirrors.

dtzWill commented:
Updated and rebased to resolve conflicts (after ncurses change to always use ncurses6 source but with different ABI's).

dtzWill committed:
ncurses 6.1: fix URL, accomodate captoinfo/infotocap along with tic

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=6c9f0af7-21da-4acf-b140-4dac9750c36a)

<details><summary>Partial log (click to expand)</summary><p>

```
rmdir: failed to remove '/nix/store/5scfwj07hab52rzwc0vld1aa0jvm8mjc-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/5scfwj07hab52rzwc0vld1aa0jvm8mjc-ncurses-6.1-dev/bin/tabs to /nix/store/j5fwvjw7digisrbrlgfcclzbidf6jnsq-ncurses-6.1/bin/tabs
rmdir: failed to remove '/nix/store/5scfwj07hab52rzwc0vld1aa0jvm8mjc-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/5scfwj07hab52rzwc0vld1aa0jvm8mjc-ncurses-6.1-dev/bin/tic to /nix/store/j5fwvjw7digisrbrlgfcclzbidf6jnsq-ncurses-6.1/bin/tic
rmdir: failed to remove '/nix/store/5scfwj07hab52rzwc0vld1aa0jvm8mjc-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/5scfwj07hab52rzwc0vld1aa0jvm8mjc-ncurses-6.1-dev/bin/tput to /nix/store/j5fwvjw7digisrbrlgfcclzbidf6jnsq-ncurses-6.1/bin/tput
rmdir: failed to remove '/nix/store/5scfwj07hab52rzwc0vld1aa0jvm8mjc-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/5scfwj07hab52rzwc0vld1aa0jvm8mjc-ncurses-6.1-dev/bin/tset to /nix/store/j5fwvjw7digisrbrlgfcclzbidf6jnsq-ncurses-6.1/bin/tset
rmdir: failed to remove '/nix/store/5scfwj07hab52rzwc0vld1aa0jvm8mjc-ncurses-6.1-dev/bin': Directory not empty
/nix/store/j5fwvjw7digisrbrlgfcclzbidf6jnsq-ncurses-6.1
```
</p></details>

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=53cf6a84-ad92-4e01-ade3-255bddde4e46)

<details><summary>Partial log (click to expand)</summary><p>

```
rmdir: failed to remove '/nix/store/2zk7yd0w96mmjfm7hs4zjhwh6ng68bh6-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/2zk7yd0w96mmjfm7hs4zjhwh6ng68bh6-ncurses-6.1-dev/bin/tabs to /nix/store/yfnys8891ip2zcs5pbf0sm5f6a814w23-ncurses-6.1/bin/tabs
rmdir: failed to remove '/nix/store/2zk7yd0w96mmjfm7hs4zjhwh6ng68bh6-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/2zk7yd0w96mmjfm7hs4zjhwh6ng68bh6-ncurses-6.1-dev/bin/tic to /nix/store/yfnys8891ip2zcs5pbf0sm5f6a814w23-ncurses-6.1/bin/tic
rmdir: failed to remove '/nix/store/2zk7yd0w96mmjfm7hs4zjhwh6ng68bh6-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/2zk7yd0w96mmjfm7hs4zjhwh6ng68bh6-ncurses-6.1-dev/bin/tput to /nix/store/yfnys8891ip2zcs5pbf0sm5f6a814w23-ncurses-6.1/bin/tput
rmdir: failed to remove '/nix/store/2zk7yd0w96mmjfm7hs4zjhwh6ng68bh6-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/2zk7yd0w96mmjfm7hs4zjhwh6ng68bh6-ncurses-6.1-dev/bin/tset to /nix/store/yfnys8891ip2zcs5pbf0sm5f6a814w23-ncurses-6.1/bin/tset
rmdir: failed to remove '/nix/store/2zk7yd0w96mmjfm7hs4zjhwh6ng68bh6-ncurses-6.1-dev/bin': Directory not empty
/nix/store/yfnys8891ip2zcs5pbf0sm5f6a814w23-ncurses-6.1
```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=d0a29f48-78dc-4369-b7a1-0bc324a7dab1)

<details><summary>Partial log (click to expand)</summary><p>

```
Moving /nix/store/1ndb2g3qq5ahlrbrwsl240jyggbxwm3q-ncurses-6.1-dev/bin/tic to /nix/store/i3sgcc3phnl8jz3m44jj1pand4f80cf8-ncurses-6.1/bin/tic
rmdir: failed to remove '/nix/store/1ndb2g3qq5ahlrbrwsl240jyggbxwm3q-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/1ndb2g3qq5ahlrbrwsl240jyggbxwm3q-ncurses-6.1-dev/bin/tput to /nix/store/i3sgcc3phnl8jz3m44jj1pand4f80cf8-ncurses-6.1/bin/tput
rmdir: failed to remove '/nix/store/1ndb2g3qq5ahlrbrwsl240jyggbxwm3q-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/1ndb2g3qq5ahlrbrwsl240jyggbxwm3q-ncurses-6.1-dev/bin/tset to /nix/store/i3sgcc3phnl8jz3m44jj1pand4f80cf8-ncurses-6.1/bin/tset
rmdir: failed to remove '/nix/store/1ndb2g3qq5ahlrbrwsl240jyggbxwm3q-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/1ndb2g3qq5ahlrbrwsl240jyggbxwm3q-ncurses-6.1-dev/bin/captoinfo to /nix/store/i3sgcc3phnl8jz3m44jj1pand4f80cf8-ncurses-6.1/bin/captoinfo
rmdir: failed to remove '/nix/store/1ndb2g3qq5ahlrbrwsl240jyggbxwm3q-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/1ndb2g3qq5ahlrbrwsl240jyggbxwm3q-ncurses-6.1-dev/bin/infotocap to /nix/store/i3sgcc3phnl8jz3m44jj1pand4f80cf8-ncurses-6.1/bin/infotocap
rmdir: failed to remove '/nix/store/1ndb2g3qq5ahlrbrwsl240jyggbxwm3q-ncurses-6.1-dev/bin': Directory not empty
```
</p></details>

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=892deefd-377f-43cc-afbb-d5b656e1f72b)

<details><summary>Partial log (click to expand)</summary><p>

```
rmdir: failed to remove '/nix/store/ibb9y4ypw9xprs221r2bighkiq96j439-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/ibb9y4ypw9xprs221r2bighkiq96j439-ncurses-6.1-dev/bin/tput to /nix/store/4fng1kpca482m5jq2wbznahj1yypvyb1-ncurses-6.1/bin/tput
rmdir: failed to remove '/nix/store/ibb9y4ypw9xprs221r2bighkiq96j439-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/ibb9y4ypw9xprs221r2bighkiq96j439-ncurses-6.1-dev/bin/tset to /nix/store/4fng1kpca482m5jq2wbznahj1yypvyb1-ncurses-6.1/bin/tset
rmdir: failed to remove '/nix/store/ibb9y4ypw9xprs221r2bighkiq96j439-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/ibb9y4ypw9xprs221r2bighkiq96j439-ncurses-6.1-dev/bin/captoinfo to /nix/store/4fng1kpca482m5jq2wbznahj1yypvyb1-ncurses-6.1/bin/captoinfo
rmdir: failed to remove '/nix/store/ibb9y4ypw9xprs221r2bighkiq96j439-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/ibb9y4ypw9xprs221r2bighkiq96j439-ncurses-6.1-dev/bin/infotocap to /nix/store/4fng1kpca482m5jq2wbznahj1yypvyb1-ncurses-6.1/bin/infotocap
rmdir: failed to remove '/nix/store/ibb9y4ypw9xprs221r2bighkiq96j439-ncurses-6.1-dev/bin': Directory not empty
/nix/store/4fng1kpca482m5jq2wbznahj1yypvyb1-ncurses-6.1
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=61a26870-02ae-422b-889b-9a685e37e191)

<details><summary>Partial log (click to expand)</summary><p>

```
cannot build derivation '/nix/store/7m6fmhzaklm1sxbi8g680kjgx4m5h5aj-cctools-binutils-darwin.drv': 4 dependencies couldn't be built
cannot build derivation '/nix/store/jiynvdgmrln6a3sxgyjrqcmr1wr94xay-gnutar-1.30.drv': 3 dependencies couldn't be built
cannot build derivation '/nix/store/dr0lw9cx1g3p51cva3gjmx797h8mg4vv-CF-osx-10.10.5.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/ilw8h51xwwkbvbgcdn641fm04zqb4pn7-cctools-binutils-darwin-wrapper.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/ma591lxdgimqcr6wgwcym0zx77h5li97-clang-wrapper-5.0.1.drv': 9 dependencies couldn't be built
cannot build derivation '/nix/store/2a4aks9m8ra2r3yf97rlb8nc9wb27b2q-stdenv-darwin.drv': 35 dependencies couldn't be built
cannot build derivation '/nix/store/3pzmx7dyj9r27alj4ib3ddl6g3mjiljz-libiconv-osx-10.11.6.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/nf75b6d1ala2bxl388fc4f4jr3q9xm67-pkg-config-0.29.2.drv': 3 dependencies couldn't be built
cannot build derivation '/nix/store/z8qk7iz95zjgi9577vxbi1r0r1vinksy-ncurses-6.1.drv': 4 dependencies couldn't be built
error: build of '/nix/store/z8qk7iz95zjgi9577vxbi1r0r1vinksy-ncurses-6.1.drv' failed
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=bda121f3-5793-4930-8a4d-9b3f4f09716b)

<details><summary>Partial log (click to expand)</summary><p>

```
cannot build derivation '/nix/store/cxbdi89w0mf1qza154c8wjkh0iiljyba-cctools-binutils-darwin.drv': 4 dependencies couldn't be built
cannot build derivation '/nix/store/4h0z7gf1kp0d8hn63z6815vf1sqp0g6s-gnutar-1.30.drv': 3 dependencies couldn't be built
cannot build derivation '/nix/store/7gqxcb501bc40zys1d6c6q2ham5hyj9g-CF-osx-10.10.5.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/r9lrgmmmifclcca8zx6hbsj7f43i8m4l-cctools-binutils-darwin-wrapper.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/b7zxd2j6rl32k3jppbr37sx4m9kcyy8v-clang-wrapper-5.0.1.drv': 9 dependencies couldn't be built
cannot build derivation '/nix/store/09z9jb279qvhdxjivgx2riamm7dykzdc-stdenv-darwin.drv': 35 dependencies couldn't be built
cannot build derivation '/nix/store/bmnkdyrlnn1n5bb69l4vkgn9g2ilqaxi-libiconv-osx-10.11.6.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/zbm15vyd3sfd139f25cdwwbk1188c9va-pkg-config-0.29.2.drv': 3 dependencies couldn't be built
cannot build derivation '/nix/store/d8dfyb6b0n0i3gwyq1fdaiw8pavp0ih7-ncurses-6.1.drv': 4 dependencies couldn't be built
error: build of '/nix/store/d8dfyb6b0n0i3gwyq1fdaiw8pavp0ih7-ncurses-6.1.drv' failed
```
</p></details>

dezgeg commented:
> From my experience on slightly older tree,
a few old programs incorrectly rely on ncurses internal details
and may now require setting NCURSES_INTERNAL=1 to allow this badness.

Oh, interesting. Do you happen to know this build failure could be avoided with that? (It started happening ever since I couldn't figure how to get ncurses5 compiling on gcc7 and switched to "--with-abi-version=5" instead on the gcc-7 branch).

````
cc -g -O2  -DHAVE_CONFIG_H  -c mach_dep.c
mach_dep.c: In function 'lock_sc':
mach_dep.c:406:2: warning: ignoring return value of 'fgets', declared with attribute warn_unused_result [-Wunused-result]
  (void) fgets(prbuf, MAXSTR, stdin);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
gcc -g -O2  -DHAVE_CONFIG_H  -c main.c
gcc -g -O2  -DHAVE_CONFIG_H  -c mdport.c
mdport.c: In function 'md_hasclreol':
mdport.c:264:17: error: dereferencing pointer to incomplete type 'TERMINAL {aka struct term}'
     if (cur_term->type.Strings == NULL)
                 ^~
make: *** [Makefile:130: mdport.o] Error 1
````

https://hydra.nixos.org/build/69789236

dezgeg commented:
Anyway, assuming that it works on Darwin, sounds good to me. I think this is just hitting ofborg timeouts, so could e.g. @LnL7 take a look of this on Darwin?

dtzWill commented:
> Oh, interesting. Do you happen to know this build failure could be avoided with that? (It started happening ever since I couldn't figure how to get ncurses5 compiling on gcc7 and switched to "--with-abi-version=5" instead on the gcc-7 branch).

Yes! rogue is the one I remember, there was another but I don't remember.

From a random local patch:

```
NIX_CFLAGS_COMPILE = [ "-DNCURSES_INTERNALS=1" ];
```

also: https://trac.macports.org/ticket/54171 .

Oh hey, the other one was "squishyball".  Not sure if needed on current nixpkgs.

See: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=860334

dtzWill commented:
re:squishyball, I remember now how I managed to not send that upstream.

Here's a rebased branch with the fixes: https://github.com/dtzWill/nixpkgs/tree/fix/squishyball

However the patch URL's are from a reworked debian-patches script..... and even so are likely unstable :(.

Anyway just sharing in case someone comes looking for whatever it was I was referring to :).

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=60bb073d-6013-48dd-9432-352a809f516f)

<details><summary>Partial log (click to expand)</summary><p>

```
rmdir: failed to remove '/nix/store/bvizlynzc03nwsbybnrm4bfbnnpzdn2x-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/bvizlynzc03nwsbybnrm4bfbnnpzdn2x-ncurses-6.1-dev/bin/tput to /nix/store/s3iamcqh1mamlpz5af9zxr739kkpw117-ncurses-6.1/bin/tput
rmdir: failed to remove '/nix/store/bvizlynzc03nwsbybnrm4bfbnnpzdn2x-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/bvizlynzc03nwsbybnrm4bfbnnpzdn2x-ncurses-6.1-dev/bin/tset to /nix/store/s3iamcqh1mamlpz5af9zxr739kkpw117-ncurses-6.1/bin/tset
rmdir: failed to remove '/nix/store/bvizlynzc03nwsbybnrm4bfbnnpzdn2x-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/bvizlynzc03nwsbybnrm4bfbnnpzdn2x-ncurses-6.1-dev/bin/captoinfo to /nix/store/s3iamcqh1mamlpz5af9zxr739kkpw117-ncurses-6.1/bin/captoinfo
rmdir: failed to remove '/nix/store/bvizlynzc03nwsbybnrm4bfbnnpzdn2x-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/bvizlynzc03nwsbybnrm4bfbnnpzdn2x-ncurses-6.1-dev/bin/infotocap to /nix/store/s3iamcqh1mamlpz5af9zxr739kkpw117-ncurses-6.1/bin/infotocap
rmdir: failed to remove '/nix/store/bvizlynzc03nwsbybnrm4bfbnnpzdn2x-ncurses-6.1-dev/bin': Directory not empty
/nix/store/s3iamcqh1mamlpz5af9zxr739kkpw117-ncurses-6.1
```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=bc94eb33-8f63-4505-a128-9a14402a7983)

<details><summary>Partial log (click to expand)</summary><p>

```
rmdir: failed to remove '/nix/store/8mr33139q4gz3h0lhkn8z4dis2v86cyw-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/8mr33139q4gz3h0lhkn8z4dis2v86cyw-ncurses-6.1-dev/bin/tput to /nix/store/3jpx6dv36q4qzav7ga8qvrgl1d4k3gfc-ncurses-6.1/bin/tput
rmdir: failed to remove '/nix/store/8mr33139q4gz3h0lhkn8z4dis2v86cyw-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/8mr33139q4gz3h0lhkn8z4dis2v86cyw-ncurses-6.1-dev/bin/tset to /nix/store/3jpx6dv36q4qzav7ga8qvrgl1d4k3gfc-ncurses-6.1/bin/tset
rmdir: failed to remove '/nix/store/8mr33139q4gz3h0lhkn8z4dis2v86cyw-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/8mr33139q4gz3h0lhkn8z4dis2v86cyw-ncurses-6.1-dev/bin/captoinfo to /nix/store/3jpx6dv36q4qzav7ga8qvrgl1d4k3gfc-ncurses-6.1/bin/captoinfo
rmdir: failed to remove '/nix/store/8mr33139q4gz3h0lhkn8z4dis2v86cyw-ncurses-6.1-dev/bin': Directory not empty
Moving /nix/store/8mr33139q4gz3h0lhkn8z4dis2v86cyw-ncurses-6.1-dev/bin/infotocap to /nix/store/3jpx6dv36q4qzav7ga8qvrgl1d4k3gfc-ncurses-6.1/bin/infotocap
rmdir: failed to remove '/nix/store/8mr33139q4gz3h0lhkn8z4dis2v86cyw-ncurses-6.1-dev/bin': Directory not empty
/nix/store/3jpx6dv36q4qzav7ga8qvrgl1d4k3gfc-ncurses-6.1
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34477&attempt_id=fa8d9fac-15c7-4075-ac3b-9455b4923473)

<details><summary>Partial log (click to expand)</summary><p>

```
cannot build derivation '/nix/store/zxnn99svlf9v6fa8kvb97918r609bkqq-cctools-binutils-darwin.drv': 4 dependencies couldn't be built
cannot build derivation '/nix/store/vx50103swvcdmhz8gvbg8aj114pk88ri-gnutar-1.30.drv': 3 dependencies couldn't be built
cannot build derivation '/nix/store/0xf05kq6kjnkn7qsrc2k1pbxp9yl63z5-CF-osx-10.10.5.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/dy1fqsllsii95hq07wijacz5yln1xgrl-cctools-binutils-darwin-wrapper.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/4bnqcg4qm6b440w74i9w3rvprvvbqs26-clang-wrapper-5.0.1.drv': 9 dependencies couldn't be built
cannot build derivation '/nix/store/6klpzkg42ik557195k563zm5r76sdi6z-stdenv-darwin.drv': 35 dependencies couldn't be built
cannot build derivation '/nix/store/z689fa6lk02wlv7vgdh1kz94w7q4di5r-libiconv-osx-10.11.6.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/sddwz2zaslwm2rmn74imngy8jz2x3v8f-pkg-config-0.29.2.drv': 3 dependencies couldn't be built
cannot build derivation '/nix/store/7n0aj867jp6p9hc1gxbvl6b23jkp9c4s-ncurses-6.1.drv': 4 dependencies couldn't be built
error: build of '/nix/store/7n0aj867jp6p9hc1gxbvl6b23jkp9c4s-ncurses-6.1.drv' failed
```
</p></details>
